src/views/QuestionView.vue

```diff
@@ -216,24 +222,14 @@ export default {
           this.filters.countryFilter !== "en:world"
             ? this.filters.countryFilter
             : null,
-          count
+          count,
+          this.page
         )
         .then((result) => {
           this.remainingQuestionCount = result.data.count;
-          if (result.data.questions.length == 0) {
-            if (!this.questionBuffer.includes(NO_QUESTION_LEFT)) {
-              if (clean) {
-                this.questionBuffer = [NO_QUESTION_LEFT];
-              } else {
-                this.questionBuffer = [
-                  ...this.questionBuffer,
-                  NO_QUESTION_LEFT,
-                ];
-              }
-            }
-            return;
+          if (this.remainingQuestionCount / count + 1 < this.page) {
+            this.page = 1;
```

Comment:
If you are out of range you do not want to set page back to 0, but to add a `No_Remaining_Questions`

Reply:
I added that condition if somehow the user reaches some page out of range. No remaining questions is added only if the number of total questions remaining becomes less than the page size i.e. 10